build.gradle

```diff
@@ -35,9 +36,7 @@ buildscript {
 
         isSameMajorVersion = opensearch_version.split("\\.")[0] == bwcVersionShort.split("\\.")[0]
         swaggerVersion = "2.1.24"
-        jacksonVersion = "2.18.1"
-        swaggerCoreVersion = "2.2.27"
-
+        swaggerCoreVersion = "2.2.28"
```

Comment:
Can remove this line as https://github.com/opensearch-project/flow-framework/pull/1005 is merged.